tokio-console/src/view/task.rs

```diff
@@ -68,76 +67,172 @@ impl TaskView {
             })
             .collect();
 
+        let stats_area_check = Layout::default()
+            .direction(layout::Direction::Horizontal)
+            .constraints(
+                [
+                    layout::Constraint::Percentage(50),
+                    layout::Constraint::Percentage(50),
+                ]
+                .as_ref(),
+            )
+            .split(area);
+
+        let location_heading = "Location: ";
+        let location_max_width = stats_area_check[0].width as usize - 2 - location_heading.len(); // NOTE: -2 for the border
+        let max_width_stats_area = area.width - 45;
+        let mut location_lines_vector: Vec<String> = task
+            .location()
+            .to_string()
+            .chars()
+            .collect::<Vec<char>>()
+            .chunks(max_width_stats_area as usize)
+            .map(|chunk| chunk.iter().collect())
+            .collect();
+        let no_of_lines_extra_required_to_accomadate_location = location_lines_vector.len() - 1;
```

Comment:
These very long variable names don't really fit in with the style of the rest of the codebase, we probably don't need the extra variable at all.

Instead, why don't we extract out the task stats height into a variable:

```suggestion
        // Id, Name, Target, Location (multiple), total, busy, scheduled, and idle times + top/bottom borders
        let task_stats_height = 9 + location_lines_vector.len();
```